CI/before_script.msvc.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
```

Comment:
Is this supposed to still be here?  
Perhaps it could at least be a `[ -n "$CI" ] && set -x` in that case?

We probably don't want to spam people with incomprehensible data when they just want to get a development environment set-up after all.